### ISSUE TITLE
Tinkerbell ci pipeline (pr-tinkerbell) not triggering.

### ISSUE THREAD

**Issue by ipochi** [kinvolk/lokomotive#1399]
It didnt trigger in this PR: (this has tinkerbell changes) https://github.com/kinvolk/lokomotive/pull/1374

Another PR specifically for tinkerbell, still not triggered
https://github.com/kinvolk/lokomotive/pull/1398

Another pipeline, which is master-tinkerbell is triggered correctly, once the Pr is merged.

**Comment by ipochi** [comment]
This was caused due to several PRs in pending state and hence new ones weren't triggering due to max in-flight setting.
